models/db.py

```diff
@@ -1,9 +1,36 @@
-from gluon.tools import Auth
 db = DAL('sqlite://storage.sqlite',pool_size=1,check_reserved=['all'])
+from gluon.tools import Auth, Service, PluginManager
+
 auth = Auth(db)
 
+## This allows me to still be able to use the web2py auth module while still customising properly the fields
```

Comment:
First, please use PEP8 :) "Each line of a block comment starts with a # and a single space" https://www.python.org/dev/peps/pep-0008/#comments

Second, a personal opinion: I try to keep comments impersonal. I would rephrase it to: "the table allows using the web2py auth module, while customising the fields"